format.html

```diff
@@ -0,0 +1,83 @@
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <div>
+        <ul>
+            <li>Sunday</li>
+            <li>Monday</li>
+            <li>Tuesday</li>
+            <li>Wednesday</li>
+            <li>Thrusday</li>
```

Comment:
add proper css properties to each list according to design for days list it should have square bullet
`.days {
    list-style: square;
}`

Reply:
got i sir